system-test/system_test_yaml_config.sh

```
@@ -0,0 +1,1065 @@
+        sed -i.bak 's|PROMETHEUS_CONFIG_FILE=/app/config/basic.yml|PROMETHEUS_CONFIG_FILE=/app/config/redis.yml|' "$COMPOSE_FILE"
+
+        # Start services
+        docker-compose -f "$COMPOSE_FILE" up -d --build
+
+        # Wait for services to be ready
+        print_status "INFO" "Waiting for services to start..."
+        sleep 15
+
+        # Wait for Redis to be ready
+        print_status "INFO" "Waiting for Redis to be ready..."
+        for i in {1..30}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)